composer.json

```diff
@@ -38,10 +39,16 @@
     "seed": "php bin/seed",
     "db:migrate": "@php bin/doctrine --no-interaction migrations:migrate",
     "db:diff": "@php bin/doctrine --no-interaction migrations:diff",
-    "test": "@php bin/phpunit"
+    "test": "@php bin/phpunit",
+    "pre-autoload-dump": "Google\\Task\\Composer::cleanup"
```

Comment:
not sure what this does?

Reply:
This command is removing all the google libraries concerning the other APIs than those we need : Calendar in this case.